### DISCUSSION TITLE
Why do the Free Worlds oufitters sell Nerve Gas Grenades

### DISCUSSION THREAD

**Discussion by ixPost** [endless-sky/endless-sky#10869] [Q&A]
The abo^e

**Comment by Hecter94** [comment]
Sales are determined by what outfitter types a planet has, not what their government is.
Nerve Gas is only sold on planets with a "Pirate Basics" or "Pirate Advanced" outfitter. If there's a Free Worlds planet with one of those outfitters, it will sell Nerve Gas. 

See, https://github.com/endless-sky/endless-sky/blob/bae0b9c269ccfcd310188cd45707a45be117d298/data/human/sales.txt#L552-L560 and https://github.com/endless-sky/endless-sky/blob/bae0b9c269ccfcd310188cd45707a45be117d298/data/human/sales.txt#L567-L577

This means that 17 planets sell nerve gas: Albatross, Bloodsea, Buccaneer Bay, Covert, Deadman's Cove, Freedom, Gagarin, Greenrock, Haven, Jakobsen, Mordente-Bridi, New Tortuga, Smuggler's Den, Stormhold, Thule, Trinket and Zenith.

**Reply by Quotatron3000** [reply]
This might be technically accurate, but from a storyline and gameplay consistency standpoint it is surprising.

During the start of Endless Sky, before Free Worlds declare freedom, the planets are deemed to be Republic and have Republic ships. The Republic prohibits nerve gas, and yet it is sold on the planet. We also see a lot of IMS ships before Free Worlds have officially become a thing.

After Free Worlds declares independence, the planets get marked Free Worlds. Now we could try to suggest the Free Worlds are heavily libertarian and therefore don't enforce Republic laws, but one of their major complaints about the Republic was the fact they didn't enforce law and order against pirates. So why would the Free Worlds tolerate pirates selling nerve gas?

I think in-game there needs to be an explanation for the discrepancy (I.E. the Free World hates pirates but love guns), or have nerve gas removed as part of a wider effort to make boarding harder.

**Reply by ixPost** [reply]
Well apparently the pirates are selling It undercover.

**Comment by ziproot** [comment]
Yeah, if you are talking about Trinket, the description mentions pirate vendors (emphasis mine):
Many of the starships parked here have scorch marks and dents on their hulls, a sure sign of recent battle. **It is likely that some of the people walking through the port are in fact pirates, posing as civilians in order to sell stolen cargo or to take a vacation incognito.**

**Reply by ixPost** [reply]
But does the Militia actually scan you, cause they're always saying they're looking out for sus ships?
